source/tutorials/fmt-cdiff-khanna.rst

```diff
@@ -506,6 +587,58 @@ We're also often interested in performing a differential abundance test at a spe
 .. question::
    Which genera differ in abundance across Subject? In which subject is each genus more abundant?
 
+
+Using q2-sample-classifier to predict sample characteristics
+------------------------------------------------------------
+
+Looks like CDI leads to a state of microbial dysbiosis. Could we use features of that dysbiosis state to diagnose CDI? Let's use :doc:`q2-sample-classifier <sample-classifier>` to find out. We will train a Random Forest classifier on a subset of our samples, and test the predictive accuracy of this model using another subset of our samples to determine how well this model generalizes to unseen samples. We will train our model to distinguish between pre- and post-FMT samples as a proxy measurement of current CDI:
+
+.. command-block::
+
+  qiime sample-classifier classify-samples \
+    --i-table disease-table.qza \
+    --m-metadata-file sample_metadata.tsv \
+    --m-metadata-column day_since_fmt \
+    --p-n-estimators 200 \

```

Comment:
Can we add what `--p-n-estimators 200 \ --p-random-state 666 \` are and why we picked those values?

Reply:
I have added a link to the sample classifier tutorial to give info on what those parameters are; it would be too verbose and redundant to describe those parameters and the meaning of those settings here.